generic3g/specs/VariableSpec.F90

```diff
@@ -111,6 +117,12 @@ function new_VariableSpec( &
       _SET_OPTIONAL(bracket_size)
       _SET_OPTIONAL(dependencies)
 
+      ! regridding parameter
+      var_spec%regrid_param = EsmfRegridderParam() ! use default regrid method
+      regrid_method = get_regrid_method_from_field_dict(var_spec%standard_name)
```

Comment:
This does not handle the case where the dictionary does not have a regrid method.   Do we need to handle an RC?   We have a default in the line above, so it would seem that we should allow the dictionary to have missing items.

Have to run.  But we can discuss later how to address this.

Reply:
I think we can handle this once NUOPC FieldDictionary is available for use.